src/System.Device.Gpio/System.Device.Gpio.csproj

```diff
@@ -1,16 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <RuntimeIdentifiers>win;linux</RuntimeIdentifiers>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <LangVersion>preview</LangVersion>
-    <!-- Only produce reference assembly in one of our inner builds -->
-    <ProduceReferenceAssembly Condition="'$(TargetsLinux)' == 'true'">true</ProduceReferenceAssembly>
```

Comment:
I agree that we won't need a ref assembly any longer, but please also remove the TargetRefPath property after this line since that won't be used either any longer